advanced-achievements-plugin/src/main/resources/config.yml

```diff
@@ -277,16 +277,15 @@ Places:
           Display: Become the Stone God!
 
 # When a specific block is broken (specify material name, lower case).
-# Below, the notation sand:1 means sand item with metadata 1 (red sand).
 Breaks: 
-  sand:1:
+  red_sand:
     10:
       Message: "&cYou have collected 10 red sand blocks!"
-      Name: break_10_sand1
+      Name: break_10_red_sand
       DisplayName: Red Sand Fan
   # Use the following construct to track multiple types. Note that if ever you add/remove a type grouped to others with |,
   # progress for this sub-category group will start back at 0.
-  sand|sand:1:
+  sand|red_sand:
```

Comment:
I believe "ice", "snow_block" and "packed_ice" have stayed the same. We could possibly use them together for some sort of "Winter Is No Longer Coming" achievement to emphasise the fact that we're destroying a lot of ice. Probably a ridiculous idea, but hey, that's what's crossing my mind right now. 😄